html/screensharingtest.js

```diff
@@ -511,8 +511,23 @@ function newRemoteFeed(id, display) {
 			onlocaltrack: function(track, on) {
 				// The subscriber stream is recvonly, we don't expect anything here
 			},
-			onremotetrack: function(track, mid, on) {
-				Janus.debug("Remote track (mid=" + mid + ") " + (on ? "added" : "removed") + ":", track);
+			onremotetrack: function(track, mid, on, metadata) {
+				Janus.debug(
+					"Remote track (mid=" + mid + ") " +
+					(on ? "added" : "removed") +
+					(metadata? " (" + metadata.reason + ") ": "") + ":", track
+				);
+				// Screen sharing tracks are sometimes muted/unmuted by browser
+				// when data is not flowing fast enough, this can make streams blink,
+				// we can ignore these
+				if(
+					track.kind === "video"
+					&& metadata
+					&& (metadata.reason === "mute" || metadata.reason === "unmute")
+				) {
```

Comment:
Code style: please put the conditions on the same line as the `if`. In case it ends up being too long, we do a carriage return with two tabs, so that it's "more" indented than the action.